packages/main/src/FileUploader.ts

```diff
@@ -254,6 +254,24 @@ class FileUploader extends UI5Element implements IFormElement {
 		}
 	}
 
+	_ondrag(e: DragEvent) {
+		e.preventDefault();
+		e.stopPropagation();
+	}
+
+	_ondrop(e: DragEvent) {
+		this._ondrag(e);
```

Comment:
Why do we call _ondrag here? Personally I would pref if we "double" the code than calling the drag handler in the drop handler just to call preventDefault and stopPropagation. If we really want to escape duplication, we can have a function, which i think is overkill in that scenario, as preventDefault and stopPropagation are methods that are usually repeated in different handlers.

Reply:
Maybe it's not worth it to shorten the code with a single line this way.
Done.